homeassistant/components/overkiz/number.py

```diff
@@ -97,6 +97,28 @@ async def _async_set_native_value_boost_mode_duration(
         max_value_state_name=OverkizState.CORE_MAXIMAL_SHOWER_MANUAL_MODE,
         entity_category=EntityCategory.CONFIG,
     ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_TARGET_DWH_TEMPERATURE,
+        name="Target Temperature",
```

Comment:
Same as other PR:
```suggestion
        name="Target temperature",
```
([It's specified](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/) that "Device, Area, and Entity names all start with a capital letter, the rest of the words are lower case")